app/models/user.rb

```diff
@@ -21,22 +21,31 @@ class User < ActiveRecord::Base
 
   # Scopes #####################################################################
 
-  scope :active,   -> { where("status = 'active'").
-                        sort_by { |u| [u.last_name, u.first_name] } }
-  scope :alumni,   -> { where("status = 'alumni'").
-                        sort_by { |u| [u.last_name, u.first_name] } }
-  scope :pending,  -> { where("status = 'pending'").
-                        sort_by { |u| [u.last_name, u.first_name] } }
+  scope :active, -> { where("status = 'active'").
+                      sort_by { |u| [u.last_name, u.first_name] }
+  }
+  scope :alumni, -> { where("status = 'alumni'").
+                      sort_by { |u| [u.last_name, u.first_name] }
+  }
+  scope :pending, -> { where("status = 'pending'").
+                       sort_by { |u| [u.last_name, u.first_name] }
+  }
   scope :inactive, -> { where("status = 'inactive'").
-                        sort_by { |u| [u.last_name, u.first_name] } }
+                        sort_by { |u| [u.last_name, u.first_name] }
+  }
+  scope :not_pending, -> { where.not("status = 'pending'") }
+  scope :org_chart_dummy, -> { where("status = 'org_chart_dummy'") }
+  scope :unassigned_to_org_chart, -> { where.not(status: 'pending').
```

Comment:
Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.